# Chromatin loop anchors predict exon usage
The codes here are used to explore if the chromatin loop information, i.e. ChIA-PET data, together with other epigenomics and transcriptomics data could contribute to the transcription and exon usage prediction.  
The datasets can be found at: https://researchdata.ntu.edu.sg/dataverse/chrom_pred_exon.

## PUBLICATION

## EXPLANATION

 
```
utils_lncRNA.py -- functions will be used.  
DeepCPP.ipynb -- user interface.  



## WORKING MECHANISM OF DeepCPP
The overview of the whole pipeline illustrated in Figure 1.
![alt text](https://github.com/mjflab/exon-prediction/blob/main/method.pdf)
figure 1.An overview of the pipeline.

## USAGE:
Based on python2.  
tested on Linux

Python modules:  
```
numpy  
pandas  
sklearn
```


## REFERANCE

## CONTACT
If you have any inqueries, please contact mfullwood@ntu.edu.sg.
